docs/resources.rst

```diff
@@ -8,508 +8,10 @@ Resources for Computing Travel Cost
-    <tr>
-      <th class="tg-1wig">Setting</th>
-      <th class="tg-1wig">Tract / Block</th>
-      <th class="tg-1wig"><img src="https://github.com/GeoDaCenter/GeoDaCenter.github.io/raw/master/images/walking.svg" height="42" width="42"/></th>
-      <th class="tg-1wig"><img src="https://github.com/GeoDaCenter/GeoDaCenter.github.io/raw/master/images/transit.svg" height="42" width="42"/></th>
-      <th class="tg-1wig"><img src="https://github.com/GeoDaCenter/GeoDaCenter.github.io/raw/master/images/driving.svg" height="42" width="42"/></th>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Max Travel Time</td>
-      <td class="tg-0lax">Tract</td>
-      <td class="tg-0lax">Inf</td>
-      <td class="tg-0lax">Inf</td>
-      <td class="tg-0lax">Inf</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Max Walking Dist.</td>
-      <td class="tg-0lax">Tract</td>
-      <td class="tg-0lax">5 km</td>
-      <td class="tg-0lax">5 km</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Walking Speed</td>
-      <td class="tg-0lax">Tract</td>
-      <td class="tg-0lax">~3 mph</td>
-      <td class="tg-0lax">~3 mph</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Departure Time</td>
-      <td class="tg-0lax">Tract</td>
-      <td class="tg-0lax">N/A</td>
-      <td class="tg-0lax">Noon, nearest Monday</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Max Travel Time</td>
-      <td class="tg-0lax">Block</td>
-      <td class="tg-0lax">90 min.</td>
-      <td class="tg-0lax">90 min.</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Max Walking Dist.</td>
-      <td class="tg-0lax">Block</td>
-      <td class="tg-0lax">5 km</td>
-      <td class="tg-0lax">5 km</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Walking Speed</td>
-      <td class="tg-0lax">Block</td>
-      <td class="tg-0lax">~3 mph</td>
-      <td class="tg-0lax">~3 mph</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-    <tr>
-      <td class="tg-0lax">Departure Time</td>
-      <td class="tg-0lax">Block</td>
-      <td class="tg-0lax">N/A</td>
-      <td class="tg-0lax">Noon, nearest Monday</td>
-      <td class="tg-0lax">N/A</td>
-    </tr>
-  </table>
-  <p class="table_center">
-    Created by: <a href="https://github.com/dfsnow">Dan Snow</a>, MPP | Sept. 2019
-  </p>
-
-|
+Since computing travel times is computationally expensive and non-trivial at scale, we've pre-computed times between common Census geographies for several travel modes. These times cover the entire United States and the most recent Census years (2020+). They are available via [OpenTimes](https://opentimes.org/), a dedicated website created by research assistant Dan Snow (UChicago MPP'19). For more information on how these times are calculated, visit the [OpenTimes GitHub](https://github.com/dfsnow/opentimes).
```

Comment:
```suggestion
Since computing travel times is computationally expensive and non-trivial at scale, we've pre-computed times between common Census geographies for several travel modes. These times cover the entire United States and the most recent Census years (2020+). They are available via [OpenTimes](https://opentimes.org/), a dedicated website created by Dan Snow (UChicago MPP'19). For more information on how these times are calculated, visit the [OpenTimes GitHub](https://github.com/dfsnow/opentimes).
```